demo/actions.py

```diff
@@ -25,6 +26,10 @@
 
 logger = logging.getLogger(__name__)
 
+# RASA_X_HOST = "rasa-x:5002"
+
+RASA_X_HOST = "localhost:5002"
+
```

Comment:
Let's make this a config parameter in demo/config.py. then we can set it in the docker-compose override